merlin-framework-processor/src/main/java/gov/nasa/jpl/aerie/merlin/processor/MissionModelParser.java

```diff
@@ -409,6 +413,17 @@ class (old-style) or as a record (new-style) by determining
         effectModel);
   }
 
+  private void checkForStaticParameters(String activityName, List<ParameterRecord> parameters) throws InvalidMissionModelException {
+    for (final var parameter: parameters) {
+      if (parameter.element.getModifiers().contains(Modifier.STATIC)) {
+        throw new InvalidMissionModelException(
```

Comment:
I think there's a way to have the error point directly to a code location... but I don't see any examples of that in this class, so no changes requested.